tiny-build/Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN apk add --no-cache \
     jq \
     curl
 
+RUN apk add --update --no-cache openssh
```

Comment:
Why not just add it after `curl` (line 33). Do we require the --update flag?

Reply:
I added --update to make sure it always adds in the latest version

Reply:
That's the default behavior of `apk add` command unless the application (i.e. `openssh`) is already installed.

Reply:
Sure, done